src/Stores/Politicians/actions.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+import type { Dispatch, PoliticianType } from '../../types';
+import { getPoliticians as apiGetPoliticians } from '../../utils/api';
+
+const getPoliticians = () => ({ type: 'POLITICIANS_LOAD' });
```

Comment:
> Am I right in assuming that this is used to display a loading message when calling the "get politicians" endpoint?

Not entirely. _Horrible explanation incoming. Consider reading the [official docs](http://redux.js.org/docs/advanced/ExampleRedditAPI.html) instead_
Actions in Redux are used to "start" or "end" things.
These three actions should be considered as more internal. It's `fetchPoliticians` that's exposed to the React components.

The "loading state" is set in the reducer https://github.com/veritus/veritus-web/pull/46/files/c6ac78956d564813de0073c5cc75ee713d501f89#diff-44f5154eb361e2c32b5d793ca5fd7f30R16

This loading state can be used for other things besides displaying a loading message.
- Prevent other HTTP calls to the same endpoint
- Call other HTTP endpoints we know need to be loaded

I highly recommend Mark Eriksons' [repo](https://github.com/markerikson/react-redux-links) for guides on this subject.
It might be a lot to take in at once, but I'm happy to pick out something specific for you if you want